File: app/src/androidTest/java/com/example/benchmarks/models/benchmark/TestOperationsMaps.java
package com.example.benchmarks.models.benchmark;

public class TestOperationsMaps extends OperationsMaps {
    private static final long MEASURE_TIME = 100L;
    @Override
    public long measureTime(int sizeOfCollection, BenchmarkItem item) {
        try {
            Thread.sleep(500);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        return MEASURE_TIME;
    }
}

File: app/src/androidTest/java/com/example/benchmarks/models/benchmark/TestOperationsMaps.kt
package com.example.benchmarks.models.benchmark

class TestOperationsMaps : OperationsMaps() {
    override fun measureTime(sizeOfCollection: Int, item: BenchmarkItem): Long {
        try {
            Thread.sleep(Tests.DELAY)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
        return Tests.MEASURE_TIME
    }
}